apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html

```diff
@@ -71,153 +109,195 @@
                                     </li>
                                 </ul>
                             </div>
-                        </div>
-                        <button class="btn btn-default desktop-view-btn mr-r15" [tooltip]="commonLocaleData?.app_export"
-                            (click)="exportCsvDownload()" [ladda]="exportInvoiceRequestInProcess$ | async"
-                            *ngIf="selectedVoucher === 'sales' && selectedInvoicesList.length && isExported"
-                            data-spinner-size="20px">
-                            <span class="icon-export-icon"></span>
-                        </button>
-                        <!-- E-way bill generation and export to excel for a single selected item in invoice ends sizes -->
-
-                        <!-- Section to create New Sales Invoice, Cash Invoice, Credit Note and Debit Note -->
-                        <div class="export-button-wrap" *ngIf="selectedVoucher === 'sales'">
-                            <button class="btn btn-primary"
-                                [disabled]="((selectedInvoices && selectedInvoices.length > 0) ? selectedInvoices.length : voucherData && voucherData.totalItems) === 0"
-                                (click)="openModal(bulkExport)">{{commonLocaleData?.app_export}}</button>
-                        </div>
-                        <div class="btn-group desktop-view-btn" dropdown *ngIf="!isCompany && selectedVoucher === 'sales'"
-                            [autoClose]="true">
-                            <button *ngIf="!isCompany" id="ddl-newinv" dropdownToggle type="button"
```

Comment:
Do check this ngIf should not be removed as we don't allow the user to create or edit in company mode